pkgs/development/python-modules/python-openstackclient/default.nix

```diff
@@ -73,6 +75,8 @@ buildPythonPackage rec {
         python-heatclient
         python-ironicclient
         python-manilaclient
+        python-neutronclient
+        python-octaviaclient
```

Comment:
```suggestion
```

Sorry I said a mistake, it cannot work with python-octaviaclient currently.
Because we have an infinite recursion here when building openstackclient-full which depends on python-octaviaclient which depends on openstackclient, which depends on python-octaviaclient...